### PR TITLE
Show all/successful projects

### PR DESCRIPTION
**Pull request by jpopelka**
This is just a proof of concept - to show that we have such data.

### CODE REVIEW

**Comment by jpopelka** [comment]
Snímek z 2020-01-04 18-53-55

File: templates/projects.html [hunk 1]
```diff
@@ -1,3 +1,7 @@
-<h2> Total project Counter {{ counter }} </h2>
+<h2> Total projects {{ counter }} </h2>
 
-{{ projects|safe }}
\ No newline at end of file
+{{ all_projects|safe }}
```
Comment (lachmanfrantisek) [inline templates/projects.html]: What about this:

```suggestion
{% for repo in all_projects %}
<div class="pf-c-card">
    <div class="pf-c-card__header pf-c-title pf-m-md">{{repo}}</div>
    <div class="pf-c-card__body">https://project.url and other info</div>
</div>
{% endfor %}
```

You need to remove `str` conversion in `render_template` call.

Comment (lachmanfrantisek) [reply templates/projects.html]: btw, we have some probably some "{repository_name}" value in redis:

Screenshot from 2020-01-05 12-04-50

File: main.py [hunk 2]
```diff
@@ -44,17 +44,6 @@ def return_json(url, method="GET", **kwargs):
     return output
 
 
-def return_json_all_pages(url, limit=10, method="GET", **kwargs):
```
Comment (jscotka) [inline main.py]: why this method is removed? I think that the method that you wrote ``all_builds`` could use some generic ones, like this, and just change URL, becasuse lots of pages there are paginated, so it needs to read all pages for complete information.

Comment (jpopelka) [reply main.py]: fixed - there's more generic `all_from()` function now

**Comment by jpopelka** [comment]
@lachmanfrantisek nice suggestions, however, this was really meant as a proof of concept to show just the **number** of projects rather than some fancy table. It certainly needs a complete redesign, but that's up to some other PR/work. 

> I vote to merge anything useful immediately

+1, merging now

**Pull request merged**